### PR TITLE
Making the about form example work as expected. Using Enter inside an…

### PR DESCRIPTION
**Pull request by NathanWalker**
Hey guys, since the first time I encountered this seed project, I felt this was needed.
I made the about form example work as expected. Using `Enter` inside an input to submit a form meets baseline usability expectations. The Add button works as well.

I think people should be seeded with a setup that at least meets basic usability expectations. I thought if this would be considered something for the showcase app, but I think this is a fundamental expectation :+1:

### CODE REVIEW

File: app/components/about/about.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
 <p>
   You want more? Add them yourself!
 </p>
-<p>
+<form (submit)="addName(newname); $event.preventDefault();">
```
Comment (ludohenin) [inline app/components/about/about.html]: I think `return false;` on the addName method should prevent submit default behaviour.
That would look a little bit prettier.

Comment (NathanWalker) [inline app/components/about/about.html]: Agreed. Thanks!

**Comment by NathanWalker** [comment]
Thanks @ludohenin, I updated the PR, added a comment in the `addName` method as well as modified the return `type`.

**Comment by mgechev** [comment]
Makes sense. Merging.

**Pull request merged**